tests/unit/pot/nhpot.cpp

```diff
@@ -39,7 +39,7 @@ ZTEST(pot, test_constexpr_NHPOT)
 	zassert_equal(0, NHPOT(UINT64_MAX));
 
 	zassert_equal(64, val);
-	zassert_equal(64, nhpot(42));
+	zassert_equal(64, nhpot(42ULL));
```

Comment:
Should this be 
```suggestion
	zassert_equal(64ULL, nhpot(42ULL));
```

as well then if we want to be using "correct" types?

Reply:
sounds good, done